### PR TITLE
Feature: APP-518 - Store list of favorited DAOs in localstorage

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- reads favorited DAOS from localStorage
- changes underlying type to allow for appropriate display on explore page

Task: [APP-518](https://aragonassociation.atlassian.net/browse/APP-518)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-518]: https://aragonassociation.atlassian.net/browse/APP-518?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/containers/navbar/index.tsx [hunk 1]
```diff
@@ -70,7 +71,7 @@ const Navbar: React.FC = () => {
   const {open} = useGlobalModalContext();
   const {pathname} = useLocation();
   const {isDesktop} = useScreen();
-  const {methods, isConnected} = useWallet();
+  const {methods, isConnected, chainId} = useWallet();
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/navbar/index.tsx]: Why use the wallet's `chainID`?

**Review by WissenIstNacht — COMMENTED** [review]
Thanks, just got one comment.

File: packages/web-app/src/containers/navbar/index.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/containers/navbar/index.tsx]: Mistake on my end. I didn't think about the wallet not being connected.

Comment (WissenIstNacht) [reply packages/web-app/src/containers/navbar/index.tsx]: Just as a reminder: It's not only that. The chain info should always be coupled with the DAO (which in turn comes from the URL), not the wallet. Basically, the wallet is ignored, unless it's required by the app. For example when signing something, or when gating the user from some flows.

**Review by WissenIstNacht — APPROVED** [review]

**Pull request merged**